wicketd/src/rss_config.rs

```diff
@@ -121,6 +122,32 @@ impl CurrentRssConfig {
         if self.external_certificates.is_empty() {
             bail!("at least one certificate/key pair is required");
         }
+
+        // When certs were uploaded we might not have known our external DNS
+        // name, so we skipped validating hostnames. We now _do_ know our
+        // external DNS name, so repeat validation with that hostname.
+        let mut cert_validator = CertificateValidator::default();
+        cert_validator.danger_disable_expiration_validation();
+
+        // This _requires_ all certs have wildcard entries; see
```

Comment:
I don't think we want to validate that here because pre-3163 this _is_ only used for the Recovery Silo.  I think we just want to validate that it matches the recovery Silo's DNS name.  If we add fleet-wide certificates later, that will still be a valid check (though not the best check we could make).

(sorry for the confusion on this)